pkg/config/config_template.yaml

```diff
@@ -1738,6 +1738,20 @@ api_key:
 #
 # dogstatsd_stats_enable: false
 
+## @param dogstatsd_log_file - string - optional
```

Comment:
I would lean towards leaving these two config options out of the default template, these feel like they'll be very infrequently used config options. 

It could be useful to add a note saying that the stats will be written to a log file in the above `dogstatsd_stats_enable` block though. Users may not want to have metric names/tags visible on disk, and it's not obvious that the `stats_enable` will now do that.

Reply:
I removed the 2 env vars in the yaml file and added comment on how enabling the `dogstatsd_stats_enable` env var will now generate dogstatsd metric logs